floris/tools/floris_interface.py

```diff
@@ -630,13 +658,18 @@ def get_turbine_ais(self) -> NDArrayFloat:
         turbine_ais = axial_induction(
             velocities=self.floris.flow_field.u,
             yaw_angle=self.floris.farm.yaw_angles,
+            tilt_angle=self.floris.farm.tilt_angles,
+            ref_tilt_cp_ct=self.floris.farm.ref_tilt_cp_cts,
             fCt=self.floris.farm.turbine_fCts,
+            tilt_interp=self.floris.farm.turbine_fTilts,
+            correct_cp_ct_for_tilt=self.floris.farm.correct_cp_ct_for_tilt,
             turbine_type_map=self.floris.farm.turbine_type_map,
         )
         return turbine_ais
 
     @property
     def turbine_average_velocities(self) -> NDArrayFloat:
+        # TODO: Should we have a `turbine_effective_velocities` function?
```

Comment:
Will this be used in analysis? If so, it would be good to have.

Reply:
@rafmudaf Unsure if it would be used or not. It isn't currently, that I know of. The `turbine_average_velocities` are used in a couple of examples, but it doesn't include the effect of yaw or tilt (which would be included in the rotor effective velocity). @paulf81 What do you think?